lib/graphql/schema/validation.rb

```diff
@@ -100,7 +100,8 @@ def self.assert_named_items_are_valid(item_name, get_items_proc)
 
         DEFAULT_VALUE_IS_VALID_FOR_TYPE = -> (type) {
           if !type.default_value.nil?
-            coerced_value = type.type.coerce_input(type.default_value)
+            coerced_value = type.type.coerce_result(type.default_value)
```

Comment:
Previously we took a stringified default value and turned it into an application value. Now, don't we have an application value _already_? Or, why do we need to `coerce_result` on it?